subprojects/docs/src/docs/userguide/authoring-builds/build_lifecycle.adoc

```diff
@@ -57,76 +72,82 @@ include::{snippetsPath}/buildlifecycle/basic/tests/buildlifecycle.out[]
+Then, Gradle evaluates the settings file. Finally, Gradle instantiates `Project` instances for each project.
 
-* It looks for `settings.gradle` in parent directories.
-* If not found, the build is executed as a single project build.
-* If a `settings.gradle` file is found, Gradle checks if the current project is part of the multi-project hierarchy defined in the found `settings.gradle` file.
-If not, the build is executed as a single project build. Otherwise a multi-project build is executed.
+=== Detect Settings File
 
-What is the purpose of this behavior? Gradle needs to determine whether the project you are in is a subproject of a multi-project build or not.
-Of course, if it is a subproject, only the subproject and its dependent projects are built, but Gradle needs to create the build configuration for the whole multi-project build (see <<multi_project_configuration_and_execution.adoc#configuration_and_execution,Configuration and Execution>>).
-If the current project contains a `settings.gradle` file, the build is always executed as:
+When you run Gradle in a directory that contains a `settings.gradle` file, Gradle uses that `settings.gradle` file to initialize the build.
+You can run Gradle within any subproject of the build.footnote:[Gradle supports partial multi-project builds (see <<intro_multi_project_builds.adoc#intro_multi_project_builds,Executing Multi-Project Builds>>).]
+When you run Gradle in a directory that contains *no* `settings.gradle` file:
 
-* a single project build, if the `settings.gradle` file does not define a multi-project hierarchy
-* a multi-project build, if the `settings.gradle` file does define a multi-project hierarchy.
+. Gradle looks for a `settings.gradle` file in parent directories.
+. If Gradle finds a `settings.gradle` file, Gradle checks if the current project is part of the multi-project build. If so, Gradle builds as a multi-project.
+. If Gradle does not find a `settings.gradle` file, Gradle builds as a single project.
 
-The automatic search for a `settings.gradle` file only works for multi-project builds with a default project layout where project paths match the physical subproject layout on disk.
-Gradle supports arbitrary physical layouts for a multi-project build, but for such arbitrary layouts you need to execute the build from the directory where the settings file is located.
-For information on how to run partial builds from the root, see <<intro_multi_project_builds.adoc#sec:executing_tasks_by_fully_qualified_name,Executing tasks by their fully qualified name>>.
+In the standard Gradle project layout, project paths match the physical subproject layout on disk.
+The automatic search for a `settings.gradle` file only works for multi-project builds with a standard project layout.
+To build projects that use a nonstandard layout, execute the build from the directory that contains `settings.gradle`.
```

Comment:
This also works for me locally. Do you know who we can ask to find out if and where this breaks? I'm not comfortable removing this statement without some guarantee that we now better support nonstandard layouts. But I suspect there are *some* nonstandard layouts that would break settings file detection.